src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala

```diff
@@ -220,6 +227,27 @@ object MCSUtils {
   }
 
   /**
+   * Write nodes to hdfs
+   * @param hdfsDir  String The hdfs directory
+   * @param localDir The local directory
+   * @param filename The filename to be copied
+   */
+  def copyNodesToHDFS(hdfsDir: String, localDir: String, filename: String): Unit = {
```

Comment:
This is an awesome addition.
I understand why naming it writeNodeToHDFS makes sense within the context of MCSUtils.

Could we have this function's code go in a function named copyNetCDFToHDFS in here: https://github.com/SciSpark/SciSpark/blob/master/src/main/scala/org/dia/utils/NetCDFUtils.scala?

We can still keep the copyNodesToHDFS function, but it would call copyNetcdfToHDFS in NetCDFUtils.scala.

What do you think @kwhitehall ?